src/dialogs/notifications/notification-drawer.js

```diff
@@ -139,7 +139,19 @@ export class HuiNotificationDrawer extends EventsMixin(
       .filter((entityId) => computeDomain(entityId) === "configurator")
       .map((entityId) => hass.states[entityId]);
 
-    return notificationsBackend.concat(configuratorEntities);
+    const notifications = notificationsBackend.concat(configuratorEntities);
+
+    notifications.sort(function (n1, n2) {
+      if (new Date(n1.created_at) > new Date(n2.created_at)) {
+        return -1;
+      }
+      if (new Date(n1.created_at) < new Date(n2.created_at)) {
```

Comment:
Don't do the date parsing twice, store it in a variable.

Reply:
done